utilities.js

```diff
@@ -0,0 +1,317 @@
+// *** FOR JEST TESTING ***
+
+// Helper function for matching input objects with objects returned from DB queries
+const matchesDatabase = (inputObject, databaseObject) => {
+    let areAllPropertiesMatched = true;
```

Comment:
this is a little concerning - prefer to assume they don't match and then determine if they do. Defaulting to true could lead to some scary bugs that would render all of your tests inaccurate

Reply:
Done